src/utils/nanvix-bench/src/main.rs

```diff
@@ -269,6 +269,15 @@ impl Benchmark {
             if ret_code < 0 {
                 error!("error sending SIGINT to nano VM: {}", std::io::Error::last_os_error());
             }
+
```

Comment:
(nitpick) Could you avoid `unwrap`s in this function by using `if let Some( .. ) = ` pattern?